src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -555,6 +625,23 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
         }
     }
 
+    public static Map<String, String> getDefaultProperties(String filePath, String moduleName)
+            throws CheckstyleException {
+
+        final String fullyQualifiedClassName = getFullyQualifiedClassName(filePath, moduleName);
+        final ModuleDetails moduleDetails = MODULE_DETAILS_MAP.get(fullyQualifiedClassName);
+
+        List<ModulePropertyDetails> properties = moduleDetails.getProperties();
+        if (properties == null) {
```

Comment:
`Condition 'properties == null' is always 'false' `

<img width="643" alt="image" src="https://github.com/user-attachments/assets/ed7b8b29-76ca-428d-b5c4-50e7f95c5307" />


how can this CI been green?